submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+#### What about storing the steam?
+
+There is some debate in the community about whether steam storage is worthwhile. It is worth noting that it is not necessary for normal plant operation, since a correct ratio of exchangers to turbines (with *just* more than enough turbines) would ensure any steam produced is being consumed while the plant is running at full capacity. Meanwhile, adding optional components for steam storage, especially at a large scale, complicates your plant design. More importantly, it impacts the UPS by adding more fluid calculations. The size of this impact is arguably small compared to that of heat exchangers and steam turbines, based on my examinations and [these](https://www.reddit.com/r/technicalfactorio/comments/qc0npz/the_impact_of_nuclear_power_plants_on_ups/) [two](https://www.reddit.com/r/factorio/comments/s2h64o/100gw_nuclear_setup_ups_comparison_2x2_345ms_vs/) detailed explorations by technical community members (further discussed below). Meanwhile, steam storage offers some optional benefits but it is up to you to decide if you prefer these benefits at the cost of some extra space and UPS.
+
+In the early game when U-235 is slow to obtain, steam storage prevents fuel waste: When your power demand is low, excess steam builds up in the system. If the steam system is full, the boilers stop and the heat starts building up in the reactors, which provide an additional energy buffer up until 1000°C. After that, the reactors continue to accept and burn fuel cells no matter what, wasting them. A steam storage setup prevents this scenario, and it is even more effective if you configure your uranium fuel cell inserters to activate only when the steam storage is running low (you can do this with a single red/green cable between each inserter and a steam storage tank). However, after unlocking the Kovarex Enrichment Process, uranium fuel cells become very easy to obtain and so letting the reactors heat up and dump heat is not a big concern any longer. However, you might still want to keep the steam storage after Kovarex if you want to keep your reactors below 900°C for safety reasons.
```

Comment:
```suggestion
In the early game when U-235 is hard to obtain, steam storage prevents fuel waste: When your power demand is low, excess steam builds up in the system. If the steam system is full, the boilers stop and the heat starts building up in the reactors, which provide an additional energy buffer up until 1000°C. After that, the reactors continue to accept and burn fuel cells no matter what, wasting them. A steam storage setup prevents this scenario, and it is even more effective if you configure your uranium fuel cell inserters to activate only when the steam storage is running low (you can do this with a single red/green cable between each inserter and a steam storage tank). However, after unlocking the Kovarex Enrichment Process, uranium fuel cells become very easy to obtain and so letting the reactors heat up and dump heat is not a big concern any longer. However, you might still want to keep the steam storage after Kovarex if you want to keep your reactors below 900°C for safety reasons.
```

Reply:
You might want to replace `In the early game when U-235 is slow to obtain` with `In the early game when U-235 is hard to obtain`